teknologiradar.json

```diff
@@ -163,6 +163,21 @@
         }
       ]
     },
+    {
+      "key": "Spring-framework",
+      "title": "Spring Framework",
+      "ring": "avstå",
+      "quadrant": "programmering",
+      "description": "Rammeverk for utvikling av backendtjenester. Vurder Micronaut før man tar i bruk Spring på nye tjenester. Brukes av SUP og Freg",
```

Comment:
Skal vi her si noe om hvem som bruker det? Da har vi plutselig en avhengighet mellom komponenter og tech rader.
Også: Burde vi si noe om hvorfor? (altså hvorfor avstå/ta i bruk/utforske, osv)

Reply:
I tilfelle avstå, men mer som en guideline. Det er ikke mulig å si at alle skal avstå fra Spring Framework særlig om for team allerede bruker det.

Reply:
> Burde vi si noe om hvorfor? (altså hvorfor avstå/ta i bruk/utforske, osv)

Ja kan skrive noe på det.

Reply:
@johnksv det skaper egentlig ikke en konkret kobling til en komponent, det er tenkt som en peker til hvor man kan evt. lære om andres erfaringer med teknologien. Jeg tror det kan være nyttig.

Oslo Origo har en innegyd "Brukt av" seksjon som en del av deres beskrivelser:
![Screenshot 2024-05-28 at 09 54 59](https://github.com/statisticsnorway/teknologiradar/assets/42948872/6b4ec014-9493-4d6b-b673-b0c2bf53209c)